lib/structures/Member.js

```diff
@@ -159,6 +155,10 @@ class Member extends Base {
         }
     }
 
+    get game() {
+        return this.activities ? this.activities[0] : null;
```

Comment:
Small thing and it doesn't really matter but should we check if the array is empty?

Reply:
it will just returns undefined. Activities is there but it's empty. 🤷🏻‍♂️